ddtrace/contrib/fastapi/patch.py

```diff
@@ -35,12 +37,34 @@ def traced_init(wrapped, instance, args, kwargs):
     wrapped(*args, **kwargs)
 
 
+async def traced_serialize_response(wrapped, instance, args, kwargs):
+    """Wrapper for fastapi.routing.serialize_response function.
+
+    This function is called on all non-Response objects to
+    convert them to a serializable form.
+
+    This is the wrapper which calls ``jsonable_encoder``.
+
+    This function does not do the actual encoding from
+    obj -> json string  (e.g. json.dumps()). That is handled
+    by the Response.render function.
+
+    DEV: We do not wrap ``jsonable_encoder`` because it calls
+    itself recursively, so there is a chance the overhead
+    added by creating spans will be higher than desired for
+    the result.
+    """
+    with ddtrace.tracer.trace("fastapi.serialize_response"):
```

Comment:
hmm not great that we can't replace the tracer instance used here but we also don't install `Pin`s on fastapi.

Not sure if we really need to do this now since it looks like we're able to test it ok without but just a note that this isn't consistent with our other integrations.

Reply:
we could pin the `fastapi` module itself to avoid using the global tracer, but then that doesn't help the trace middleware since it doesn't know about `fastapi`

Reply:
Yeah pinning to the module might be the way to go just so that it is somewhat configurable